src/card-queue/components/ItsAMatchOverlay.js

```diff
@@ -59,23 +62,65 @@ const style = {
 
 type ItsAMatchOverlayProps = {
   open: boolean,
-  avatarImgUrl2?: string,
-  avatarImgUrl1?: string,
+  avatarImgOwn?: number,
+  avatarImgTarget?: number,
   muiTheme: any,
-  targetName?: string
+  targetUserName: string,
+  targetFirstName: string,
+  targetLastName: string,
+  onContinue?: () => void,
+  onStartMessaging?: () => void
 }
+
 class ItsAMatchOverlay extends Component {
   props: ItsAMatchOverlayProps;
   render () {
-    const {avatarImgUrl1, avatarImgUrl2, open} = this.props
+    const {
+      avatarImgOwn,
+      avatarImgTarget,
+      open,
+      onContinue = () => {},
+      onStartMessaging = () => {}
+    } = this.props
     const { palette } = this.props.muiTheme
+    const {
+      targetFirstName,
+      targetLastName
+    } = this.props
     return (
       <Dialog open={open} contentStyle={style.filter}>
-        <h3 style={{...style.primaryText, color: palette.primary2Color}}>It's a match!</h3>
-        <h4 style={style.secondaryText}>You and {this.props.targetName || 'someone'} have liked each other's trip!</h4>
+        <h3
+          style={{
+            ...style.primaryText,
+            color: palette.primary2Color
+          }}>
+          It's a match!
+        </h3>
+        <h4 style={style.secondaryText}>
+          You and {
+            targetFirstName + targetLastName ||
+            'someone'
+          } have liked each other's trip!
+        </h4>
         <div>
-          <Avatar className='animated ease slideInLeft' style={style.avatar} size={128} src={avatarImgUrl1 || stockAvatar} />
-          <Avatar className='animated ease slideInRight' style={style.avatar} size={128} src={avatarImgUrl2 || stockAvatar} />
+          <Avatar
+            className='animated ease slideInLeft'
+            style={style.avatar}
+            size={128}
+            src={
+              avatarImgOwn === undefined
+              ? stockAvatar
+              : oidToUrl(avatarImgOwn)
+            } />
+          <Avatar
+            className='animated ease slideInRight'
+            style={style.avatar}
+            size={128}
+            src={
+              avatarImgTarget === undefined
+              ? stockAvatar
+              : oidToUrl(avatarImgTarget)
```

Comment:
It is a proper name indeed. All the images are stored as numbers, which is oid. The function converts an image's oid to an actual url